datastore/longhorn.go

```diff
@@ -50,6 +50,19 @@ func (s *DataStore) GetSetting() (*longhorn.Setting, error) {
 	return result, err
 }
 
+func (s *DataStore) GetCredentialSecret(secretName string) (map[string]string, error) {
```

Comment:
It's better to call it `GetCredentialFromSecret`

Secret is a k8s object to store secret things. Credential is the key to some kind of service. CredentialSecret is the k8s secret which stored the credential.